app/javascript/components/layouts/footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { AppProvider, FooterHelp, Link } from '@shopify/polaris'
+
+class Footer extends React.Component {
+  render () {
+    return (
+      <AppProvider>
```

Comment:
Are we sure AppProvider is required here? I thought it was supposed to surround the entirety of the app, once.

Reply:
I havent dug into why but when missing it complained it was needed and didn't render.